tw-tasks-core/src/main/java/com/transferwise/tasks/dao/ITaskDao.java

```diff
@@ -93,6 +93,8 @@ public long getVersion() {
 
   int getStuckTasksCount(ZonedDateTime age, int maxCount);
 
+  List<Pair<String, Integer>> getStuckTasksCountGrouped(ZonedDateTime age, int maxCount);
```

Comment:
The `List<Pair<?, ?>>` makes me think that the result can have pairs with duplicate keys, which is not the case. So curious why it's not `Map<String, Integer>`? The naming could be also more specific, like `getStuckTasksCountByTaskType`